sparkplug_b/sparkplug_b.proto

```diff
@@ -80,7 +80,7 @@ message Payload {
         }
 
         optional string version         = 1;          // The version of the Template to prevent mismatches
-        repeated Metric metrics         = 2;          // Each metric is the name of the metric and the datatype of the member but does not contain a value
+        repeated Metric metrics         = 2;          // Each includes the name of the metric, the datatype of the member, and optionally contains a value
```

Comment:
Change to:
"Each metric includes a name, datatype, and optionally a value"

Reply:
Changed per your suggestion.